design/ui/pagination/src/Pagination.test.js

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { cleanup, fireEvent, render } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
```

Comment:
@lordpixel FYI we do not have to import extend expect anymore. It is part of our Jest config. Auto imports it for us. Just FYI. No need to change here for now.